chart/templates/server-deployment.yaml

```diff
@@ -161,9 +161,9 @@ spec:
         - name: THEIA_PORT
           value: "{{ .Values.components.workspace.ports.http.containerPort | toString }}"
         - name: THEIA_IMAGE_REPO
```

Comment:
**question**: Shall we open a follow up issue to remove this later?

Reply:
Yes, we should create a follow-up issue to clean up the code if we don't have it yet.